Sync/RequestExtensions.swift

```diff
@@ -7,18 +7,48 @@ import Alamofire
 import Shared
 
 extension Request {
-    public func responseParsedJSON(completionHandler: ResponseHandler) -> Self {
-        return response(serializer: Request.ParsedJSONResponseSerializer(), completionHandler: { (request, response, JSON, error) in
+    public func responseParsedJSON(partial: Bool, completionHandler: ResponseHandler) -> Self {
```

Comment:
I see that we're always passing in true for partial whenever we call this. Is there a reason we would not want the JSON serializer to handle fragments by default?